components/contributors/BadgeIcons.jsx

```diff
@@ -19,15 +20,32 @@ function useOnClickOutside(ref, handler) {
   }, [ref, handler]);
 }
 export default function BadgeIcons({ skill }) {
-  const [showModel, setShowModel] = useState();
+  const [showModel, setShowModel] = useState(false);
   const ref = useRef();
   useOnClickOutside(ref, () => setShowModel(false));
+
+  const currentLevel = skill.levels.indexOf(skill.currentLevel) + 1;
+  let glow = "";
+
+  switch (skill.levels.length - currentLevel) {
+    case 0:
+      glow = "glow-gold";
+      break;
+
+    case 1:
+      glow = "glow-silver";
+      break;
+
+    default:
+      break;
+  }
+
```

Comment:
this would be better as a function, some thing like
```suggestion
 const glow = () => {
    const currentLevel = skill.levels.indexOf(skill.currentLevel) + 1;

    switch (skill.levels.length - currentLevel) {
      case 0:
        return "glow-gold";
  
      case 1:
        return "glow-silver";
  
      default:
        return "";
    }
 }

```